sros/docker/launch.py

```diff
@@ -400,6 +451,57 @@ def bootstrap_spin(self):
 
         return
 
+    def configure_power(self, power_cfg):
+        """
+        Configure power shelf/ves and modules
+        """
+        shelves = power_cfg["shelves"] if "shelves" in power_cfg else 1
+        modules = power_cfg["modules"]
+        if type(modules) is dict:
+            modules = modules[self.power]  # 3(AC) or 4(DC)
+
+        if self.power == "dc":  # vSIM default
+            power_shelf_type = f"ps-a{modules}-shelf-dc"
+            module_type = "ps-a-dc-6000"
+        else:
+            power_shelf_type = f"ps-b{modules}-shelf-ac/hv"
+            module_type = "ps-b-ac/hv-6000"
```

Comment:
the AC power modules are used for which systems, considering that vSIMs only accept DC?